lib/styles/index.css

```diff
@@ -0,0 +1,485 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&family=Kumbh+Sans:wght@400;700&family=Lato:ital,wght@0,400;0,700;1,400&display=swap");
+
+@font-face {
+  font-family: "cocogoose";
+  src: url("fonts/cocogoose/");
+}
+
+:root {
+  --grey: #d3d3d3;
+  --black: #272a31;
```

Comment:
The pages use the exact same colors, for text, backgrounds, borders, etc, from the original design 🔥 Well done!